docs/link_handling.md

```diff
@@ -48,15 +48,8 @@ Sample iOS universal link metadata
 	"applinks": {
 		"details": [
 			{
-				"appIDs": [
-					"com.google.test.application"
-				],
-				"components": [
-					{
-						"/": "/*",
-						"comment": "handle all urls"
-					}
-				]
+				"appID": "com.google.test.application",
```

Comment:
The `appID` is missing the Team ID prefix:
> The value of the appID key is the team ID or app ID prefix, followed by the bundle ID

You may want update the mobile app form to include a field for the Team ID, or mention that it should be included in the Application ID